GetStream/stream-chat-android
# [2679] Add user reactions bottom sheet

https://github.com/GetStream/stream-chat-android/issues/2679

The previous PR which was declined due to the amount of conflicts with the "develop" branch:
https://github.com/GetStream/stream-chat-android/pull/2743

### 🎯 Goal

Handle clicks in the reactions section of a message and show a menu with the list of user reactions.

### 🎨 UI Changes

https://user-images.githubusercontent.com/9600921/146791730-183ebea8-9c78-45dd-9e54-bac9f0dfd6ac.mov

### 🧪 Testing

1. Open the message list screen
2. Top on the reaction section of a message
3. Observe a bottom sheet with the latest reactions

### ☑️Contributor Checklist

#### General

#### Code & documentation
~~- [ ] New code is covered by unit tests~~
~~- [ ] Affected documentation updated (KDocs, docusaurus, tutorial)~~

### ☑️Reviewer Checklist

### 🎉 GIF

![giphy](https://user-images.githubusercontent.com/9600921/146793440-983b2830-7cff-4572-aa99-202ef56a2591.gif)
